setup.py

```diff
@@ -126,7 +126,7 @@
       description=DESCRIPTION,
       scripts=scripts,
       requires=['astropy', 'requests', 'keyring', 'beautifulsoup4',
-                'html5lib', 'six'],
+                'html5lib', 'six', 'pyvo'],
```

Comment:
`pyvo`, I envision will be an optional dependency for a while, or shall we go ahead and require is from the begining?

@keflavich @eteq?